pinot-common/src/main/java/org/apache/pinot/common/Utils.java

```diff
@@ -145,4 +149,36 @@ public static void logVersions() {
 
     return componentVersions;
   }
+
+  public static String getGitInfo() {
```

Comment:
why is this in pinot-common if this is a controller only API?Do you expect to extend the api to the other components as well?

Reply:
This follows the pattern of the version endpoint pretty much line for line. I even considered factoring out a common function to do the manifest scan but there are slight differences in logic so I decided against it.